rke/k8s_defaults.go

```diff
@@ -138,7 +163,6 @@ func GenerateData() {
 	if err := enc.Encode(TemplateData); err != nil {
 		panic(fmt.Sprintf("error encoding template data %v", err))
 	}
-	fmt.Println(buf.String())
```

Comment:
I think this was added to print out plugin information for all k8s versions to quickly see which key and range is getting used for each system image on rancher side. 

If we're removing it, could we also remove use of `TemplateData`? It's not getting used elsewhere